### ISSUE TITLE
Matching Matlab hyperparameters

### ISSUE THREAD

**Issue by AmousQiu** [JustGlowing/minisom#148]
Hi there!Thank you for this great work!

 I switched to using python from the Matlab, version of SOM However I found the result was quite different. Where I could have a perfect 100% in MatLab but somehow only get 19% in f1-score here.

The only thing I changed from the default setting in Matlab is using a 10*10. 
som = MiniSom(10, 10, 4096, sigma=1.5, learning_rate=0.7,activation_distance='euclidean',
              neighborhood_function='gaussian', topology='hexagonal', random_seed=10)
And this is what I had for my settings using minisom.

Any suggestions so I could maybe recreate the result from Matlab?

Thank you in advance!

**Comment by JustGlowing** [comment]
Hi there,

Minisom doesn't implement any classification mechanisms, hence I'm not sure how you computed the f score. There's one example in the documentation that can help you but the classification function is quite basic.

Make sure that your data is standardized and that your SOM is properly trained visualizing the distribution of the labels on the map as shown in the basic usage example.

**Comment by AmousQiu** [comment]
Hi.
I was following the classification example to get the f-score which use the same mechanism as I used before.

To check if the data is properly, I know that the visualization process but that also doesn't give me a good result. To achive a good one, is there any suggestions for adjusting the hyperparameters? 

For the standardized part, I was using TFIDF with 4k features.

**Comment by JustGlowing** [comment]
Hyperparameters tuning has already been covered here https://github.com/JustGlowing/minisom/issues/101
